logstash-core/lib/logstash/modules/kibana_config.rb

```diff
@@ -19,6 +19,7 @@ def initialize(modul, settings)
     @name = modul.module_name
     @settings = settings
     @index_name = settings.fetch("dashboards.kibana_index", ".kibana")
+    @metrics_max_buckets = settings.fetch("dashboards.metrics_max_buckets", 60000)
```

Comment:
Although I have made it so the user can alter the value in the modules section of LS settings, we need to clarify with SA team whether this is a module specific KV pair.